packages/lisk-transactions/src/0_transfer.ts

```diff
@@ -44,81 +30,48 @@ export interface TransferInputs {
 	readonly timeOffset?: number;
 }
 
-const validateInputs = ({
-	amount,
-	recipientId,
-	recipientPublicKey,
-	data,
-}: TransferInputs): void => {
-	if (!validateTransferAmount(amount)) {
-		throw new Error('Amount must be a valid number in string format.');
-	}
-
-	if (!recipientId && !recipientPublicKey) {
-		throw new Error(
-			'Either recipientId or recipientPublicKey must be provided.',
-		);
-	}
-
-	if (typeof recipientId !== 'undefined') {
-		validateAddress(recipientId);
-	}
-
-	if (typeof recipientPublicKey !== 'undefined') {
-		validatePublicKey(recipientPublicKey);
-	}
-
-	if (
-		recipientId &&
-		recipientPublicKey &&
-		recipientId !== cryptography.getAddressFromPublicKey(recipientPublicKey)
-	) {
-		throw new Error('recipientId does not match recipientPublicKey.');
-	}
-
-	if (data && data.length > 0) {
-		if (typeof data !== 'string') {
-			throw new Error(
-				'Invalid encoding in transaction data. Data must be utf-8 encoded string.',
-			);
-		}
-		if (data.length > BYTESIZES.DATA) {
-			throw new Error('Transaction data field cannot exceed 64 bytes.');
-		}
-	}
-};
-
-export const transfer = (inputs: TransferInputs): TransferTransaction => {
+export const transfer = (inputs: TransferInputs): object => {
```

Comment:
definition of `TransferInputs` is only used here, so let's move it here.
Also, return type would be probably bit better if it's Partial<TransactionJSON>.

Reply:
Both Done. Type changed to `Partial<TransactionJSON>`. Resolving.